test-cases/gemini/gemini-1tb-10h.yaml

```diff
@@ -1,4 +1,4 @@
-test_duration: 780
+test_duration: 660
```

Comment:
I don't think we should change this one.

Reply:
reverted